alma/lib/Utils/Settings.php

```diff
@@ -140,28 +140,73 @@ public static function canLog()
         return (bool) (int) self::get('ALMA_ACTIVATE_LOGGING', false);
     }
 
+    /**
+     * Get value ALMA_ACTIVATE_SHARE_OF_CHECKOUT
+     *
+     * @return string
+     */
```

Comment:
It seems `canShareOfCheckout` does not return a `bool` anymore and could change its name with `getShareOfChekcoutStatus` or something like that? WDYT?